packages-web/badge/.gitignore

```diff
@@ -0,0 +1,18 @@
+/tests/TestProjects/**/.classpath
+/tests/TestProjects/**/.project
+/tests/TestProjects/**/javascriptsource
+/tests/TestProjects/**/javasource
+/tests/TestProjects/**/resources
+/tests/TestProjects/**/userlib
+
+/tests/TestProjects/**/widgets/*
+!/tests/TestProjects/**/widgets/Badge.mpk
+!/tests/TestProjects/**/widgets/com.mendix.widget.web.Badge.mpk
```

Comment:
Should we just ignore the widgets folder?

Reply:
You got me thinking about this one. What if we just remove the whole test project from the repo. You would have to run svn checkout to get the test project or you can add your own one in the empty folder. After that you can build the widget mpk.

Reply:
mpks should not be ignored inside widgets folder

Reply:
Agree, widgets will not be ignored anymore.